.travis.yml

```diff
@@ -8,6 +8,20 @@ env:
     - ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe
     - ORACLE_SID=XE
 
+matrix:
+  include:
```

Comment:
Sounds good.

```yaml
include:
  - env: APT_ADDON=0
  - env: APT_ADDON=1
    addons: ...
```

Do we need the nearly-empty entry in the matrix or can we get by with just one? For example,

```yaml
include:
  - env: APT_ADDON=1
    addons: ...
```